spec/models/claim_spec.rb

```diff
@@ -426,6 +426,28 @@
     end
   end
 
+  describe "approved" do
+    let!(:submitted_claims) { create_list(:claim, 5, :submitted) }
+    let!(:unsubmitted_claims) { create_list(:claim, 2, :submittable) }
+    let!(:approved_claims) { create_list(:claim, 5, :approved) }
+    let!(:rejected_claims) { create_list(:claim, 5, :rejected) }
```

Comment:
Do we need to create this many claims to test this? I think we should aim to keep the number of database writes down in our test suite